.github/workflows/playwright.yml

```diff
@@ -0,0 +1,28 @@
+name: Playwright Tests
+
+on:
+  pull_request:
```

Comment:
Should this also run with staging? so we can see this before Prod

Reply:
Done - it can be running on staging now.